### ISSUE TITLE
Text between images is stripped

### ISSUE THREAD

**Issue by diplatis** [Automattic/facebook-instant-articles-wp#279]
Source Markup:
```xhtml
<p><a href="http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg"><img class="alignnone size-full wp-image-244086" src="http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?fit=565%2C753" alt="1η ΣΥΣΤΑΔΑ ΔΕΝΤΡΩΝ" srcset="http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?w=2240 2240w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=150%2C200 150w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=225%2C300 225w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=768%2C1024 768w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=364%2C485 364w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=900%2C1200 900w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=300%2C400 300w, http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1η-ΣΥΣΤΑΔΑ-ΔΕΝΤΡΩΝ.jpg?resize=565%2C753 565w" sizes="(max-width: 2240px) 100vw, 2240px" data-recalc-dims="1" /></a></p>
<p>Δενδροφάγους και επίδοξους τζιχαντιστές του πρασίνου, χαρακτηρίζει τα αιρετά στελέχη της Υπηρεσίας Πρασίνου του Δήμου Βόλου, η Περιβαλλοντική Πρωτοβουλία Μαγνησίας, με αφορμή το κλάδεμα 40 πεύκων στην Αγριά. Όπως χαρακτηριστικά αναφέρεται σε ανακοίνωση της οργάνωσης, «οι δενδροφάγοι της Υπηρεσίας Πρασίνου του Δήμου Βόλου έδρασαν πάλι, αυτή τη φορά στην Αγριά. Επιτέθηκαν ως επίδοξοι τζιχαντιστές του πρασίνου κατά πλήθους πεύκων στην οδό Τριανταφύλλου στην Αγριά. Πάνω από 40 πεύκα δέχθηκαν το «χάδι» της Υπηρεσίας Πρασίνου. Και αυτό συνίσταται στο κόψιμο όλων των κλαδιών των πεύκων εκτός από την κορυφή!!! Τα δέντρα αυτά λοιπόν έγιναν «κοψοχέρικα» και ας μη ψηφίζουν…</p>
<p><a href="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg"><img class="alignnone size-full wp-image-244089" src="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?fit=565%2C424" alt="ΚΛΑΔΕΜΑ ΠΕΥΚΟΥ!" srcset="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?w=4320 4320w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=200%2C150 200w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=300%2C225 300w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=768%2C576 768w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=485%2C364 485w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=1200%2C900 1200w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=260%2C194 260w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=565%2C424 565w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/ΚΛΑΔΕΜΑ-ΠΕΥΚΟΥ.jpg?resize=50%2C37 50w" sizes="(max-width: 4320px) 100vw, 4320px" data-recalc-dims="1" /></a><br />
Αυτή την επέλαση κατά των δέντρων οι «υπερεπιστήμονες» της Υπηρεσίας Πρασίνου την ονομάζουν «κλάδεμα». Αυτή η συμπεριφορά δεν αντέχει σε καμιά επιστημονική θεωρία και ανταποκρίνεται μονάχα σε ενδεχόμενο μίσος κατά των δέντρων. Δυστυχώς η καταστροφική μανία του Δήμου Βόλου κατά του πρασίνου της πόλης συνεχίζεται απτόητη και αυτή τη συμπεριφορά θα τη βιώσει η πόλη και με τις καλοκαιρινές ζέστες και με την ατμοσφαιρική ρύπανση του Χειμώνα. Τι να πει κανείς όμως. Οι άνθρωποι δεν καταλαβαίνουν από τίποτα. Είναι αλλού!»</p>
<p><a href="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg"><img class="alignnone size-full wp-image-244087" src="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?fit=565%2C753" alt="2η ΣΥΣΤΑΔΑ" srcset="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?w=2240 2240w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=150%2C200 150w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=225%2C300 225w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=768%2C1024 768w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=364%2C485 364w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=900%2C1200 900w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=300%2C400 300w, http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2η-ΣΥΣΤΑΔΑ.jpg?resize=565%2C753 565w" sizes="(max-width: 2240px) 100vw, 2240px" data-recalc-dims="1" /></a></p>
```

Transformed Markup:
```xml
<html>
  <head>
    <link rel="canonical" href="http://e-thessalia.gr/perivallontiki-protovoulia-dendrofagi-tou-dimou-volou-edrasan-pali/"/>
    <meta charset="utf-8"/>
    <meta property="op:generator" content="facebook-instant-articles-sdk-php"/>
    <meta property="op:generator:version" content="1.1.0"/>
    <meta property="op:generator:application" content="facebook-instant-articles-wp"/>
    <meta property="op:generator:application:version" content="2.11"/>
    <meta property="op:generator:transformer" content="facebook-instant-articles-sdk-php"/>
    <meta property="op:generator:transformer:version" content="1.1.0"/>
    <meta property="op:markup_version" content="v1.0"/>
    <meta property="fb:use_automatic_ad_placement" content="true"/>
    <meta property="fb:article_style" content="default"/>
  </head>
  <body>
    <article>
      <header>
        <figure>
          <img src="http://e-thessalia.gr/wp-content/uploads/2016/05/1&#x3B7;-&#x3A3;&#x3A5;&#x3A3;&#x3A4;&#x391;&#x394;&#x391;-&#x394;&#x395;&#x39D;&#x3A4;&#x3A1;&#x3A9;&#x39D;.jpg"/>
        </figure>
        <h1>Περιβαλλοντική Πρωτοβουλία: «Οι δενδροφάγοι του Δήμου Βόλου έδρασαν πάλι…»</h1>
        <time class="op-published" datetime="2016-05-19T17:19:34+03:00">May 19th, 5:19pm</time>
        <time class="op-modified" datetime="2016-05-19T17:19:34+03:00">May 19th, 5:19pm</time>
        <address><a>Pressroom</a></address>
        <figure class="op-ad">
          <iframe src="https://www.facebook.com/adnw_request?placement=1017033028388830_1017036365055163&amp;adtype=banner300x250" width="300" height="250"></iframe>
        </figure>
      </header>
      <figure>
        <img src="http://i2.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/1&#x3B7;-&#x3A3;&#x3A5;&#x3A3;&#x3A4;&#x391;&#x394;&#x391;-&#x394;&#x395;&#x39D;&#x3A4;&#x3A1;&#x3A9;&#x39D;.jpg?fit=565%2C753"/>
      </figure>
      <p>Δενδροφάγους και επίδοξους τζιχαντιστές του πρασίνου, χαρακτηρίζει τα αιρετά στελέχη της Υπηρεσίας Πρασίνου του Δήμου Βόλου, η Περιβαλλοντική Πρωτοβουλία Μαγνησίας, με αφορμή το κλάδεμα 40 πεύκων στην Αγριά. Όπως χαρακτηριστικά αναφέρεται σε ανακοίνωση της οργάνωσης, «οι δενδροφάγοι της Υπηρεσίας Πρασίνου του Δήμου Βόλου έδρασαν πάλι, αυτή τη φορά στην Αγριά. Επιτέθηκαν ως επίδοξοι τζιχαντιστές του πρασίνου κατά πλήθους πεύκων στην οδό Τριανταφύλλου στην Αγριά. Πάνω από 40 πεύκα δέχθηκαν το «χάδι» της Υπηρεσίας Πρασίνου. Και αυτό συνίσταται στο κόψιμο όλων των κλαδιών των πεύκων εκτός από την κορυφή!!! Τα δέντρα αυτά λοιπόν έγιναν «κοψοχέρικα» και ας μη ψηφίζουν…</p>
      <figure>
        <img src="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/&#x39A;&#x39B;&#x391;&#x394;&#x395;&#x39C;&#x391;-&#x3A0;&#x395;&#x3A5;&#x39A;&#x39F;&#x3A5;.jpg?fit=565%2C424"/>
      </figure>
      <figure>
        <img src="http://i1.wp.com/e-thessalia.gr/wp-content/uploads/2016/05/2&#x3B7;-&#x3A3;&#x3A5;&#x3A3;&#x3A4;&#x391;&#x394;&#x391;.jpg?fit=565%2C753"/>
      </figure>
      <figure class="op-tracker">
        <iframe>
          <!-- This site uses the Google Analytics by MonsterInsights plugin version 5.5 - https://www.monsterinsights.com/ -->
          <!-- @Webmaster, normally you will find the Google Analytics tracking code here, but you are in the disabled user groups. To change this, navigate to Analytics -> Settings (Ignore usergroups) -->
          <!-- / Google Analytics by MonsterInsights -->
        </iframe>
      </figure>
    </article>
  </body>
</html>```

**Comment by diegoquinteiro** [comment]
Hi,

Thanks for reporting. We'll include a fix to this bug in the next release.

Thank you,
Diego

**Comment by everton-rosario** [comment]
@diplatis This is already fixed in version 3.0+ of WP Plugin + 1.3.0 SDK
Can you double check it?
